src/stores/SSEChannel/MessagesSSEChannel.ts

```diff
@@ -156,29 +153,17 @@ export class MessagesSSEChannel extends SSEChannel<EventMessage> {
 	private _onMessageIdsEvent = (e: Event) => {
 		const messagesIdsEvent: MessageIdsEvent =
 			e instanceof MessageEvent && e.data ? JSON.parse(e.data) : null;
-		const streams = getArrayOfUniques(this.messageIds.filter(isInvalidResumeId).map(getSession));
-		this.messageIds = Object.values(messagesIdsEvent.messageIds).filter(
-			id => id && isInvalidResumeId,
+		const newIds = Object.values(messagesIdsEvent.messageIds).filter(
+			id => id && id.slice(-2) !== '-1',
 		) as string[];
-		if (streams.length > 0) {
-			api.messages
-				.getResumptionMessageIds({
-					streams,
-					startTimestamp: moment().utc().valueOf(),
-				})
-				.then(streamInfo => {
-					this.messageIds = [
-						...this.messageIds.filter(id => !isInvalidResumeId(id)),
-						...(this.queryParams.searchDirection === 'next'
-							? streamInfo.next.map(info => info.lastElement)
-							: streamInfo.previous.map(info => info.lastElement)),
-					];
-
-					if (messagesIdsEvent && this.eventListeners.onMessageIdsEvent) {
-						this.eventListeners.onMessageIdsEvent(messagesIdsEvent);
-					}
-				});
-		} else if (messagesIdsEvent && this.eventListeners.onMessageIdsEvent) {
+		this.messageIds = [
+			...newIds,
+			...this.messageIds.filter(
+				messageId =>
+					!newIds.find(id => id.includes(messageId.slice(0, messageId.lastIndexOf(':')))),
```

Comment:
There is a getSession helper function you've written, it's unused now. use it here or remove it